packages/functions/src/index.ts

```diff
@@ -20,3 +20,4 @@ export * from './https/onCall/file/getPassword';
 
 export * from './storage/onObjectFinalized/user/onHeadshotUpload';
 export * from './storage/onObjectFinalized/file/onFileUploadEncrypt';
+export * from './storage/onObjectFinalized/file/onFileUploadVirusChecks';
```

Comment:
Remove this line as it is no longer in use

Reply:
updated in latest commit